runner.py

```diff
@@ -0,0 +1,495 @@
+#!/usr/bin/env python
+
+import sys
+assert sys.version_info >= (3, 9), "Use Python 3.9 or newer."
+
+import os
+import socket
+import optparse
+import datetime
+import shutil
+import subprocess
+import time
+import signal
+import json
+from typing import Union, Tuple
+
+'''
+The default flags depending on the hostname.
+'''
+DEFAULT_SETTINGS = {
+    'futharkhpa01fl.unicph.domain': {
+        'name': 'A100',
+        'labels': 'cuda,opencl,multicore',
```

Comment:
From what I remember, the name of the runner (here A100) should also be one of the labels.  Is that automatically added elsewhere?

Reply:
I do not do this anywhere, I will add it to DEFAULT_SETTINGS.

Reply:
This should be fixed in the latest's commit.